CHANGELOG.md

```diff
@@ -3,9 +3,14 @@
 ## [0.8.0 UNRELEASED]
```

Comment:
Can you set it up under 0.7.1, because of this bug #441 (it should be fixed in this version, blocker for some hive module)?
```suggestion
## [0.7.1 UNRELEASED]
```

Reply:
Ah, yup

Reply:
I will rebase pr #443 later, after this PR will be merged